packages/datadog_flutter_plugin/test/rum/rum_user_action_detector_test.dart

```diff
@@ -148,16 +148,19 @@ void main() {
     verifyNoMoreInteractions(mockRum);
   });
 
-  testWidgets('tap gesture detector with annotation reports description',
+  testWidgets(
+      'tap gesture detector with annotation reports description and attributes',
```

Comment:
Let's leave the existing tests alone since the basic version of this API remains unchanged. Instead, let's add a new tests that checks that when attributes are added they're forwarded properly.